cellmlmanip/model.py

```diff
@@ -745,9 +743,9 @@ def remove_variable(self, symbol):
                 self.rdf.remove(triple)
 
         # Remove references to variable and invalidate cache
-        del self._name_to_symbol[symbol.name]
+        del self._name_to_variable[symbol.name]
```

Comment:
Should we also change local variable names like here `symbol` becomes `variable`?

Reply:
Done!